sklearn/gaussian_process/_gpr.py

```diff
@@ -93,12 +93,11 @@ def optimizer(obj_func, initial_theta, bounds):
         run is performed.
 
     normalize_y : boolean, optional (default: False)
-        Whether the target values y are normalized, i.e., the mean of the
-        observed target values become zero. This parameter should be set to
-        True if the target values' mean is expected to differ considerable from
-        zero. When enabled, the normalization effectively modifies the GP's
-        prior based on the data, which contradicts the likelihood principle;
-        normalization is thus disabled per default.
+        Whether the target values y are normalized, the mean and variance of
+        the target values are set equal to 0 and 1 respectively. This is
+        recommended for cases where zero-mean, unit-variance priors are used.
+        Note that, in the following, the normalisation is reversed before the
```

Comment:
What does "the following" refer to?

Reply:
Sorry I thought I'd replied to this. I just meant 'in the follow code'. Basically, trying to tell the user that the predictive results will be 'un-normalised' before being reported back.

Reply:
I don't usually assume that the reader is looking in the code file, but often at pre-compiled documentation, so "in this implementation" might be more appropriate.

Reply:
Please also add a `.. versionchanged:: 0.23` tag.